rfd/0079/project-service-draft.md

```diff
@@ -0,0 +1,121 @@
+# Mariposa Project service
+
+The Mariposa (RFD36) Project service will be responsible for maintaining and exposing data about user projects and services. The project service will expose multiple endpoints for interacting with this data, to be used by the Triton CLI as well as other Triton services, including the Convergence service. 
```

Comment:
> used by the Triton CLI as well as other Triton services

perhaps we should be more clear here that it's "exposed to the users via CloudAPI and consumed directly by services within Triton"?